sftpgo.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: sftpgo
+  version: 2.6.2
+  epoch: 0
+  description: Full-featured and highly configurable SFTP, HTTP/S, FTP/S and WebDAV server - S3, Google Cloud Storage, Azure Blob
+  copyright:
+  - license: AGPL-3.0-only
+  dependencies:
+    provides:
```

Comment:
we dont need provide here unless we do version stream, if i understand correctly.

Reply:
As enabled update, I'd like to provide version stream for next version too

Reply:
version stream as is if you want to provide multiple versions of sftpgo in wolfi repo.